visualization/index.html

```diff
@@ -302,6 +321,7 @@ <h3>ASL-LEX 2.0</h3>
 
 <script src="js/userInterface.js"></script>
 <script src="//cdnjs.cloudflare.com/ajax/libs/d3-tip/0.7.1/d3-tip.min.js"></script>
+<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js" integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM" crossorigin="anonymous"></script>
```

Comment:
Lets keep the protocol relative URL style (using //) as done in the line above for consistency